test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs

```diff
@@ -1,41 +1,45 @@
 using System.Collections.Generic;
 using Microsoft.Test.Apex.VisualStudio.Solution;
-using NuGet.StaFact;
-using Xunit;
-using Xunit.Abstractions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace NuGet.Tests.Apex
 {
-    public class NetCoreProjectTestCase : SharedVisualStudioHostTestClass, IClassFixture<VisualStudioHostFixtureFactory>
+    [TestClass]
+    public class NetCoreProjectTestCase : SharedVisualStudioHostTestClass
     {
-        public NetCoreProjectTestCase(VisualStudioHostFixtureFactory visualStudioHostFixtureFactory, ITestOutputHelper output)
-            : base(visualStudioHostFixtureFactory, output)
+        private const int Timeout = 5 * 60 * 1000; // 5 minutes
+
+        public NetCoreProjectTestCase()
+            : base()
         {
         }
 
         // basic create for .net core template
-        [NuGetWpfTheory]
-        [MemberData(nameof(GetNetCoreTemplates))]
-        public void CreateNetCoreProject_RestoresNewProject(ProjectTemplate projectTemplate)
+        [TestMethod]
```

Comment:
Do we need to change `NuGetWpfTheory` to MSTest attribute or created new one?
https://github.com/NuGet/NuGet.Client/blob/892375745964db627494305fd605496454b5ad0a/test/TestExtensions/NuGet.StaFact/NuGetWpfTheoryAttribute.cs#L10